yuuna/lib/src/pages/implementations/subtitle_options_dialog_page.dart

```diff
@@ -29,25 +33,36 @@ class _SubtitleOptionsDialogPage
   late final TextEditingController _delayController;
   late final TextEditingController _fontSizeController;
   late final TextEditingController _fontNameController;
+  late final TextEditingController _fontColorController;
+  late final TextEditingController _outLineColorController;
   late final TextEditingController _regexFilterController;
   late final TextEditingController _opacityController;
   late final TextEditingController _widthController;
   late final TextEditingController _blurController;
 
   late ValueNotifier<bool> _aboveBottomBarNotifier;
+  Color fontColor = Colors.white;
+  Color outLineColor = Colors.white;
+  List<String> fontWeights = ['Thin', 'Normal', 'Bold'];
+  int fontWeightIdx = 1;
 
   @override
   void initState() {
     super.initState();
     _options = widget.notifier.value;
 
+    fontWeightIdx = fontWeights.indexOf(_options.fontWeight);
+    fontColor = Color(_options.fontColor);
+    outLineColor = Color(_options.subtitleOutlineColor);
     _allowanceController =
         TextEditingController(text: _options.audioAllowance.toString());
     _delayController =
         TextEditingController(text: _options.subtitleDelay.toString());
     _fontSizeController =
         TextEditingController(text: _options.fontSize.toString());
     _fontNameController = TextEditingController(text: _options.fontName.trim());
+    _fontColorController = TextEditingController(text: 'Font Color');
```

Comment:
Am I correct in understanding that this will be the initial text that appears when the settings dialog appears? If so, this won't do. I would want to see the current hex value of the color here instead.